### DISCUSSION TITLE
按照指南配置完成后无法使用，新发现：仅在服务端重启v2ray后的一小段时间（5-10秒）内可以使用

### DISCUSSION THREAD

**Discussion by dan-gel** [v2fly/v2ray-core#1594] [Q&A]
一开始发现服务端上v2ray只监听了ipv6，于是在服务端配置文件中加了"listen"="ipv4".
更改重启之后发现仍然无法使用
客户端日志：
[Warning] [1130327253] app/proxyman/outbound: failed to process outbound traffic > proxy/vmess/outbound: failed to find an available destination > common/retry: [dial tcp xxx.xxx.xxx.xxx:xxx: i/o timeout dial tcp xxx.xxx.xxx.xxx:xxx: operation was canceled] > common/retry: all retry attempts failed

对应端口已在vultr上放行且在防火墙上打开，端口扫描也显示已开放
求大佬指教orz

没关闭客户端重启了服务端发现突然可以访问Google.com然后刷新一次又无法访问，可以复现

附上客户端和服务端配置文件
客户端(后来尝试用v2rayN进行配置的)：
{
"log": {
"access": "D:\vps\V2RayN\v2ray-windows-64\Vaccess.log",
"error": "D:\vps\V2RayN\v2ray-windows-64\Verror.log",
"loglevel": "warning"
},
"inbounds": [
{
"tag": "proxy",
"port": 1080,
"listen": "127.0.0.1",
"protocol": "socks",
"sniffing": {
"enabled": true,
"destOverride": [
"http",
"tls"
]
},
"settings": {
"auth": "noauth",
"udp": true
}
}
],
"outbounds": [
{
"tag": "proxy",
"protocol": "vmess",
"settings": {
"vnext": [
{
"address": "xxx.xxx.xxx.xxx",
"port": xxxx,
"users": [
{
"id": "xxxxxx",
"alterId": 0,
"email": "[t@t.tt](mailto:t@t.tt)",
"security": "aes-128-gcm"
}
]
}
]
},
"streamSettings": {
"network": "tcp"
},
"mux": {
"enabled": false,
"concurrency": -1
}
},
{
"tag": "direct",
"protocol": "freedom",
"settings": {}
},
{
"tag": "block",
"protocol": "blackhole",
"settings": {
"response": {
"type": "http"
}
}
}
],
"routing": {
"domainStrategy": "IPIfNonMatch",
"rules": [
{
"type": "field",
"inboundTag": [
"api"
],
"outboundTag": "api"
}
]
}
}

服务端：
{
"log": {
"loglevel": "warning",
"access": "/var/log/v2ray/access.log",
"error": "/var/log/v2ray/error.log"
},
"inbounds": [{
"port": xxx,
"protocol": "vmess",
"listen":"xx.xx.xx.xx",
"settings": {
"clients": [{ "id": "xxxxx" }]
}
}],
"outbounds": [{
"protocol": "freedom",
"settings": {}
}]
}

**Comment by jinlianch** [comment]
请问后面怎么解决了
